README.rst

```diff
@@ -58,6 +71,16 @@ To check if everything worked fine, you can do::
 
 and it should not give any error messages.
 
+**GUI installation**
+
+To install the GUI dependencies along with ``hnn-core``, a simple tweak to the above command is needed::
+
+   $ pip install hnn_core[gui]
```

Comment:
Could we potentially do something like this for the parallel backend dependencies as well?

Reply:
we could ... or we could have a conda-based install. I fear either route is not going to work 100% and may need constant upkeep. But if it's something you want and are willing to maintain, I am all for it ;-) See also: https://github.com/jonescompneurolab/hnn-core/pull/263/files